src/app/list/list.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore } from '@angular/fire/firestore';
+
+import { Observable } from 'rxjs';
+
+import { List } from './models/list.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ListService {
+  public lists$: Observable<List[]>;
+
+  constructor(firestore: AngularFirestore) {
+    this.lists$ = firestore.collection<List>('lists').valueChanges();
```

Comment:
We just can create a method which will return list collection

Reply:
I didn't want to have many subscriptions, but yeah, you're right, we can do this, because anyway it's going to be a hot observable.